GeosCore/RnPbBe_mod.F90

```diff
@@ -173,7 +173,7 @@ SUBROUTINE CHEMRnPbBe( Input_Opt,  State_Chm, State_Diag, &
 
```

Comment:
Might also be worth making a variable one_minus_Exp_Rn so that we don't have to repeatedly calculate it below.